CHANGELOG.md

```diff
@@ -28,12 +28,12 @@ This project adheres to
   * Write DICOM files to be more compatible with other software [Pull Request](https://github.com/pymedphys/pymedphys/pull/1578)
 * Daily Winston-lutz
   * Small workaround [Pull Request](https://github.com/pymedphys/pymedphys/pull/1565)
+* DICOM pseudonymise
+  * Mitigated susceptibility to brute-force reversal of dates and other rainbow table attacks [Pull Request](https://github.com/pymedphys/pymedphys/pull/1580)
```

Comment:
Are we wanting to have some form of guarantee that the output of DICOM pseudonymise is deterministic? Do we want to flag a change in pseudonymise outputs for the same inputs as a breaking change?

I know personally if I was relying on the outputs of pseudonymise to be deterministic (as expected by the approach) I'd appreciate a "breaking changes" flag in the release notes.

I'm happy to leave this up to you as to whether that is to form part of our "declared API" going forward. ie "same input -> same output" is part of the API spec for DICOM pseudonymise.